appium-tests/android/android.spec.js

```diff
@@ -373,7 +384,6 @@ describe('Camera tests Android.', function () {
                         return driver
                             .waitForElementByXPath('//android.widget.TextView[@text="Gallery"]', 20000)
                             .elementByXPath('//android.widget.TextView[@text="Gallery"]')
-                            .elementByXPath('//android.widget.TextView[@text="Gallery"]')
```

Comment:
Do you have any details around the emulator bug on Windows? Ideally:
 - any Appium issues filed that sound similar?
 - any Android issues filed relevant?
 - Windows version?
 - Appium version?
 - Android emulator version?

I'm also up for looking into it more closely.